docs/README-develop-types-providers.md

```diff
@@ -24,7 +24,7 @@
 
 This document is a guide for writing new Puppet resource types and providers for Cisco NX-OS.
 
-There are multiple components involved when creating new resources. This document focuses on the type and provider files:
+There are multiple components involved when creating new resources, it would be suggested to review [`custom resources`](https://puppet.com/docs/puppet/6.0/custom_resources.html) as it outlines the various ways a custom type and provider can be written for Puppet. This document focuses on the legacy type and provider files:
```

Comment:
The official wording is now "low-level", not "legacy". Please also add ", for historical context", or any other suitable emphasis on the fact that this should not be *used* anymore on new types.